torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -33,10 +33,12 @@ auto AccumulateGrad::acc_inplace(std::shared_ptr<Variable>& grad,
 
 auto AccumulateGrad::apply(const variable_list& grads) -> variable_list {
   // XXX: this method is not thread-safe!
-  check_input_variables("AccumulateGrad", grads, 1);
-  auto var = variable.lock();
+  check_input_variables("AccumulateGrad", grads, 1, 0);
   auto new_grad = grads[0];
 
+  if (!new_grad) return {};
```

Comment:
You're right, but I'm wondering when was it needed in this PR. It seems that the only situation when Convolutions return NULL grads is when weights don't require grad. But this means that they shouldn't have a GradAccumulator. Or is it triggered for bias in ConvBackwardBackward?